.gitignore

```diff
@@ -5,3 +5,5 @@ venv
 # Files
 *.md5sum
 *.env
+
+.prefectignore
```

Comment:
Not a big deal, but if you don't mind, can you throw a line break at the end of this last line? No worries if not.